### PR TITLE
Retrieves a list of RecordSets globally in the VinylDNS

### PR DESCRIPTION
**Pull request by djordan17**
Retrieves a list of RecordSets globally in the VinylDNS

### CODE REVIEW

**Comment by pauljamescleary** [comment]
Thanks @djordan17 !  Will review this today :)

File: src/main/java/io/vinyldns/java/VinylDNSClient.java [hunk 1]
```diff
@@ -358,4 +358,17 @@
...
+   * Retrieves a list of RecordSets globally in the VinylDNS database based on search criteria. A
+   * minimum of two alpha-numeric characters is required.
+   *
+   * @param request See {@link ListRecordSetGlobalRequest ListRecordSetGlobalRequest Model}
+   * @return {@link VinylDNSSuccessResponse
+   *     VinylDNSSuccessResponse&lt;ListRecordSetGlobalResponse&gt;} in case of success and {@link
+   *     VinylDNSFailureResponse VinylDNSFailureResponse&lt;ListRecordSetGlobalResponse&gt;} in case
+   *     of failure
+   */
+  VinylDNSResponse<ListRecordSetGlobalResponse> listGlobalRecordSets(
```
Comment (pauljamescleary) [inline src/main/java/io/vinyldns/java/VinylDNSClient.java:372]: This is all good, just naming.

I would call this "search" to disambiguate it from the existing `listRecordSet`

- `SearchRecordSetsRequest`
- `SearchRecordSetsResponse`
- `public VinylDNSResponse<SearchRecordSetsResponse> searchRecordSets(SearchRecordSetsRequest request)...`

Thanks again for doing this, looks great!

Comment (djordan17) [reply src/main/java/io/vinyldns/java/VinylDNSClient.java]: Renamed :)

**Comment by pauljamescleary** [comment]
Awesome, thanks @djordan17 !

**Review by pauljamescleary — APPROVED** [review]

**Pull request merged**